### ISSUE TITLE
Hover UI - shrinking dot  and clock

### ISSUE THREAD

**Issue by willwade** [asterics/AsTeRICS-Grid#38]
Would be super nice to have a different UI display for hovering:

1. A shrinking dot on the cell
2. A filling clock design

**Comment by deinhofer** [comment]
I agree. It would be a good user feedback to know the residual time until the cell will be selected.
